src/components-examples/material/select/select-harness/select-harness-example.spec.ts

```diff
@@ -0,0 +1,79 @@
+import {TestBed, ComponentFixture, waitForAsync, inject} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {MatSelectHarness} from '@angular/material/select/testing';
+import {HarnessLoader} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {MatSelectModule} from '@angular/material/select';
+import {SelectHarnessExample} from './select-harness-example';
+import {MatFormFieldModule} from '@angular/material/form-field';
+import {ReactiveFormsModule} from '@angular/forms';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {OverlayContainer} from '@angular/cdk/overlay';
+
+describe('SelectHarnessExample', () => {
+  let fixture: ComponentFixture<SelectHarnessExample>;
+  let loader: HarnessLoader;
+  let overlayContainer: OverlayContainer;
+
+  beforeAll(() => {
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+  });
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [MatSelectModule, MatFormFieldModule, ReactiveFormsModule, NoopAnimationsModule],
+        declarations: [SelectHarnessExample]
+      }).compileComponents();
+    })
+  );
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(SelectHarnessExample);
+    fixture.detectChanges();
+    loader = TestbedHarnessEnvironment.loader(fixture);
+    inject([OverlayContainer], (oc: OverlayContainer) => {
+      overlayContainer = oc;
+    })();
+  });
+
+  afterEach(() => {
+    // Angular won't call this for us so we need to do it ourselves to avoid leaks.
+    overlayContainer.ngOnDestroy();
+    overlayContainer = null!;
+  });
```

Comment:
We can drop this `afterEach` and the dependency on the `OverlayContainer` since it was added primarily for our own unit tests.